roles/php/tasks/main.yml

```diff
@@ -1,22 +1,36 @@
 ---
-- name: Add PHP 7.4 PPA
+- name: Add PHP 8.0 PPA
   apt_repository:
     repo: "ppa:ondrej/php"
     update_cache: yes
 
-- name: Install PHP 7.4
+- name: Install PHP 8.0
   apt:
     name: "{{ item.key }}"
     state: "{{ item.value }}"
     cache_valid_time: "{{ apt_cache_valid_time }}"
   with_dict: "{{ php_extensions }}"
 
-- name: Start php7.4-fpm service
+- name: Start php8.0-fpm service
   service:
-    name: php7.4-fpm
+    name: php8.0-fpm
     state: started
     enabled: true
 
+- name: Check for existing php7.4-fpm service
+  stat:
+    path: /etc/init.d/php7.4-fpm
+  register: php73_status
```

Comment:
This needs to be updated to `php74` to match the the task below

Reply:
Fixed.